extern/ZAPDUtils/Utils/StringHelper.cpp

```diff
@@ -1,7 +1,9 @@
 #include "StringHelper.h"
 
+#if (_MSC_VER)
 #pragma optimize("2", on)
 #define _CRT_SECURE_NO_WARNINGS
+#endif
```

Comment:
could you explain why this is being `#if`'d now (and why it wasn't before)?

Reply:
I can't explain why it wasn't before but I added it because that define is only needed for Windows. That pragma style from I have seen is only recognized on MSVC.